test/e2e/resources/replicationgroup_rpng.yaml

```diff
@@ -0,0 +1,15 @@
+# "RPNG" means that the field replicasPerNodeGroup is set (instead of nodeGroupConfiguration),
+# meaning that this replication group has a uniform configuration across all shards. This is one of the more
+# basic configurations as very few fields are specified and the resulting RG can either be CME or CMD
+# (depending on the number of specified node groups).
```

Comment:
Interesting. So CME or CMD basically just means whether there is more than one node group?

If this is the case, if I change an RG from 2 node groups to 1, that means the RG goes from cluster mode enabled to cluster mode disabled automagically?

Reply:
It's a little complicated. "CME" just means the RG has the ability to have more than 1 node group. It's possible to create a CME RG with 1 node group, by specifying a CME parameter group when creating the replication group (so many "groups"!). To your second question, it's also possible to scale in a CME RG from 2 node groups to 1 – but it will not change from CME to CMD.

In contrast, a CMD RG will always have 1 node group. I'm not aware if there is a way to toggle cluster mode on a RG after its creation.

I think this page should have more details about what's going on under the hood: https://redis.io/topics/cluster-tutorial